platformcomponents/desktop/participant-list.json

```diff
@@ -0,0 +1,12 @@
+{
+  "participantList": {
```

Comment:
'participantListItem' is the right naming for this component. The name of the file should also change to 'participant-list-item'

Reply:
participantListItem sounds be better.